src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/integration/documents/DossierGenerationGotenbergComponentTest.kt

```diff
@@ -44,13 +47,14 @@ class DossierGenerationGotenbergComponentTest : GotenbergComponentTestBase() {
       localDeliveryUnit = localDeliveryUnit,
       currentPrisonId = currentPrisonId
     )
-    authenticatedClient.getRecallNotification(recall.recallId)
+    authenticatedClient.generateDocument(recall.recallId, RECALL_NOTIFICATION)
     expectNoVirusesWillBeFound()
     uploadLicenceFor(recall)
     uploadPartAFor(recall)
 
-    val dossier = authenticatedClient.getDossier(recall.recallId)
+    val generateResponse = authenticatedClient.generateDocument(recall.recallId, DOSSIER)
+    val dossier = authenticatedClient.getDocument(recall.recallId, generateResponse.documentId)
     // writeBase64EncodedStringToFile("dossier-$expectedPageCount-pages.pdf", dossier.content)
```

Comment:
If you're OK to update rather than bin them then I think the `docxxx.content` in all the 
`// writeBase64EncodedStringToFile ...`
need to become `Pdf(docxxx.content)` ?

Reply:
Will do next time :)